packages/std/src/errors/mod.rs

```diff
@@ -4,6 +4,9 @@ mod system_error;
 mod verification_error;
 
 pub use recover_pubkey_error::RecoverPubkeyError;
-pub use std_error::{DivideByZeroError, OverflowError, OverflowOperation, StdError, StdResult};
+pub use std_error::{
+    ConversionOverflowError, DivideByZeroError, OverflowError, OverflowOperation, StdError,
```

Comment:
`ConversionOverflowError` also need to be exported in lib.rs because it is used in StdError now.